packages/storybook/src/DataVisualization/Stat.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react'
+import { Stat } from '@kodiak-ui/data-visualization'
+import { Box, Grid } from '@kodiak-ui/primitives'
+import { AudienceIcon } from './AudienceIcon'
+
+export default { title: 'Data Visualization/Stat' }
+
+export function StatSample() {
```

Comment:
💡 minor thing and probably okay as-is, but isn't it more standard to have each variation defined separately (rather than all of them in a grid)?

Reply:
Possibly, the variations weren't too different so it was more a test of how the api felt.  I was using the grid more to test that they would look ok the way we're going to use them in jilt initially.